java/code/src/com/suse/manager/webui/controllers/login/LoginController.java

```diff
@@ -76,6 +79,7 @@ private LoginController() { }
     public static void initRoutes(JadeTemplateEngine jade) {
         get("/manager/login", withCsrfToken(LoginController::loginView), jade);
         post("/manager/api/login", LoginController::login);
+        get("/manager/api/oidcLogin", LoginController::performOidcLogin);
```

Comment:
Have a login method using the "GET" method looks strange to me. It's doing changes on the server side, since it creates the session. 
Do you think it would make more sense to use POST instead?